test/orchard/inspect_test.clj

```diff
@@ -443,13 +474,58 @@
                inspect/next-sibling
                inspect/next-sibling
                inspect/next-sibling
-               inspect/next-sibling)))))
+               inspect/next-sibling))))
+  (testing "next and previous siblings with pagination"
+    (is (= 38 (-> long-vector
+                  inspect
+                  (inspect/down 40)
+                  (inspect/previous-sibling)
+                  :value)))
+    (is (= 38 (-> long-vector
+                  inspect
+                  (inspect/set-page-size 1)
+                  (inspect/down 40)
+                  (inspect/previous-sibling)
+                  :value)))
+    (is (= 36 (-> long-vector
+                  inspect
+                  (inspect/set-page-size 2)
+                  (inspect/down 40)
+                  (inspect/previous-sibling)
+                  (inspect/previous-sibling)
+                  (inspect/previous-sibling)
+                  :value)))
+    (is (= 40 (-> long-vector
+                  inspect
+                  (inspect/down 40)
+                  (inspect/next-sibling)
+                  :value))))
+  (testing "next sibling doesn't go beyond the current page"
+    (is (= 41 (-> long-vector
```

Comment:
Besides from this example, which is OK, please add a small, self-contained example where one can just _see_ the intended behavior.

In other words, as a maintainer it's pretty hard to build a mental model of a `(vec (range 70))` and make reasoning about it (e.g. "41 is good").

As an example that I'd find good, https://github.com/clojure-emacs/orchard/blob/11e191b3f405e7e2a2d6e4a44b42ef26499e58d8/test/orchard/inspect_test.clj#L412-L424 is a small data structure, so if I see seven `(inspect/next-sibling)` calls, it's clear to casual readers that it's trying to go beyond bounds.